ReactWindows/ReactNative.Net46/ReactNative.Net46.csproj

```diff
@@ -171,6 +171,7 @@
     <Compile Include="Views\Progress\ReactProgressBarViewManager.cs" />
     <Compile Include="Views\Scroll\ReactScrollViewManager.cs" />
     <Compile Include="Views\Slider\ReactSliderManager.cs" />
+    <Compile Include="Views\TextInput\PlaceholderAdorner.cs" />
```

Comment:
The placeholder effect is achieved by applying an adorner to the text input controls...